jwql/website/apps/jwql/data_containers.py

```diff
@@ -88,7 +89,12 @@ def get_dashboard_components():
                  'filecount_type': 'Total File Counts by Type',
                  'size_type': 'Total File Sizes by Type',
                  'filecount': 'Total File Counts',
-                 'system_stats': 'System Statistics'}
+                 'system_stats': 'System Statistics',
+                 'monitor_cron_jobs': 'Cron Job Monitor',
+                 'cron_status_table': 'Cron Job Status'}
+
+    # Run the cron job monitor to produce an updated table
+    monitor_cron_jobs.status(production_mode=False)
```

Comment:
@bhilbert4 Interesting, I wouldn't have thought to do this.  I think this is a good idea, as long as `monitor_cron_jobs.status()` doesn't take too long to run and significantly slows down the dashboard from loading on the web app.  I assume that the function is quite fast right now, but once we have a lot of log files this may eventually become a problem.  Either way, its an issue for sometime down the road I think. 

The alternative would be to remove this call completely and just rely on `cron` running `monitor_cron_jobs` (how meta) on a daily or perhaps hourly basis.  In that case, it would be possible for the table to be outdated, though hopefully not by much.

Reply:
I talked with @laurenmarietta about this a bit. It was her suggestion to put the call to `monitor_cron_jobs` in `data_containers` after I mentioned that it would be nice to have the table generated each time the dashboard is loaded. I do like the idea of the table being always up to date, but you're right that that needs to be balanced against how long it takes to run. I don't have a good intuition for how much longer it will take to run as the number of log files goes up. 

Do you want to keep it this way for now and switch to calling via cron job if the dashboard is slow later? Or should we just switch over now? Also at the moment, calling here is defaulting to `production_mode = False` while calling via command line `production_mode = True`.  I'm assuming we want to default to False for the time being?

Reply:
@bhilbert4 Yes, I agree that we should try having the call to this module in `data_containers` and we can remove it later if we find it to be burdensome.  

I think that `production_mode` should be `True` in both cases actually; I don't think seeing how often the `dev` logs are being created is that useful to know in this use case.

Reply:
Makes sense. I switched the default to True.